dynamic_programming/minimum_size_subarray_sum.py

```diff
@@ -0,0 +1,39 @@
+def minsubarraysum(target: int, numbers: list[int]) -> int:
+    """
+    Returns the length of the shortest contiguous subarray
+     in nums whose sum is at least target.
+
+    Examples :
+        >>> minsubarraysum(7, [2, 3, 1, 2, 4, 3])
+        2
+        >>> minsubarraysum(7, [2, 3, -1, 2, 4, -3])
+        4
+        >>> minsubarraysum(11, [1, 1, 1, 1, 1, 1, 1, 1])
+        0
+        >>> minsubarraysum(10, [1, 2, 3, 4, 5, 6, 7])
+        2
+        >>> minsubarraysum(5, [1, 1, 1, 1, 1, 5])
+        1
+        >>> minsubarraysum(0, [])
```

Comment:
Add tests for 0, [1, 2, 3] and for -6, []

Reply:
added.